examples/tensor/fixed_rank_prod_expressions.cpp

```diff
@@ -148,24 +148,13 @@ int main()
 
         // C2(k,l,m) = T(k,l,m) + A(i,j,k)*B(j,l,i,m) + 5;
         // Similar Problem as above
-        tensor_t C2 = dynamic_tensor(dynamic_extents<>{na[2],nb[1],nb[3]},value_t(2)) + prod(A,B,perm_t{1,2},perm_t{3,1}) + 5;
+        tensor_t C2 = dynamic_tensor<float>(dynamic_extents<>{na[2],nb[1],nb[3]},value_t(2)) + prod(A,B,perm_t{1,2},perm_t{3,1}) + 5;
 
         // formatted output
         std::cout << "% --------------------------- " << std::endl;
         std::cout << "% --------------------------- " << std::endl << std::endl;
         std::cout << "%  C2(k,l,m) = T(k,l,m) + A(i,j,k)*B(j,l,i,m) + 5;" << std::endl << std::endl;
         std::cout << "C2=" << C2 << ";" << std::endl << std::endl;
 
-
-        // C3(k,l,m) = T(k,l,m) + A(i,j,k)*trans(B(j,l,i,m),{2,3,1,4})+ 5;
-        // Similar Problem as above
-        tensor_t C3 = dynamic_tensor(dynamic_extents<>{na[2],nb[1],nb[3]},value_t(2)) + prod(A,trans(B,{2,3,1,4}),perm_t{1,2}) + 5;
-
-        // formatted output
-        std::cout << "% --------------------------- " << std::endl;
-        std::cout << "% --------------------------- " << std::endl << std::endl;
-        std::cout << "%  C3(k,l,m) = T(k,l,m) + A(i,j,k)*trans(B(j,l,i,m),{2,3,1,4})+ 5;" << std::endl << std::endl;
-        std::cout << "C3=" << C3 << ";" << std::endl << std::endl;
-
```

Comment:
this was because we don't know if extents will increase in size or not and `fixed_rank_extents` is fixed in size which will be fixed in the next pull request when I add support for static prod.

Reply:
This should not be deleted. Please provide a solution for `fixed_rank_extents` type tensors .
It should not be a superior solution but the pull request should not decrease the number of features of existed features.

Reply:
added the example with a few changes.